### DISCUSSION TITLE
homebrew is stripping the 'v' off my cli

### DISCUSSION THREAD

**Discussion by claywd** [Homebrew/discussions#4589] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

So the version of our cli is currently v2.1.4 but homebrew is stripping off the `v` which is making the cli fail and telemetry fail. How do I update this formula to stop that from happening and keep the `v` in the version? I've tried testing some different formula locally but it's made no difference thus far. I feel like it has something to do with the CI process that bumps versions which I've been unable to test locally. 

Any help or direction is appreciated. 

To reproduce this issue, 

`brew install kubefirst`
`kubefirst version`
Should show `kubefirst-cli golang utility version: v2.1.4` but instead it shows `kubefirst-cli golang utility version: 2.1.4`
https://github.com/Homebrew/homebrew-core/blob/master/Formula/kubefirst.rb

**Comment by carlocab** [comment]
You can open a pull request:
```diff
diff --git a/Formula/kubefirst.rb b/Formula/kubefirst.rb
index 3f83cf8c4fc..a4702752933 100644
--- a/Formula/kubefirst.rb
+++ b/Formula/kubefirst.rb
@@ -19,7 +19,7 @@ class Kubefirst < Formula
   depends_on "go" => :build
 
   def install
-    ldflags = "-s -w -X github.com/kubefirst/runtime/configs.K1Version=#{version}"
+    ldflags = "-s -w -X github.com/kubefirst/runtime/configs.K1Version=v#{version}"
     system "go", "build", *std_go_args(ldflags: ldflags)
 
     generate_completions_from_executable(bin/"kubefirst", "completion")
@@ -30,6 +30,6 @@ class Kubefirst < Formula
     assert_match "k1-paths:", (testpath/".kubefirst").read
     assert_predicate testpath/".k1/logs", :exist?
 
-    assert_match version.to_s, shell_output("#{bin}/kubefirst version")
+    assert_match "v#{version}", shell_output("#{bin}/kubefirst version")
   end
 end
```
Use the commit message
```
kubefirst: fix version output
```
(or similar)

**Reply by claywd** [reply]
I had that same idea, but, I made these same changes locally and when I ran `brew install --build-from-source kubefirst.rb` and it made no difference. Still had the same result. Maybe I need to do something else to test this locally? Already tapped the homebrew-core repo, made my changes there, and set the env variable to install local only formula.

**Reply by SMillerDev** [reply]
Did you build from source?

**Reply by carlocab** [reply]
Try
```
HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source kubefirst
```
In particular, **don't** use a path (`kubefirst.rb`) but a formula name (`kubefirst`).
